src/drivers/mct/cime_config/config_component_acme.xml

```diff
@@ -255,16 +255,16 @@
       <value compset="_DLND.*_MPASLI">1</value>
       <value compset="_SLND.*SOCN.*_MPASLI">1</value>
       <value compset="_DATM.*_SLND.*MPASO.*_MPASLI">24</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oQU240">12</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oQU240wLI">12</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oQU120">24</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oEC60to30">48</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oEC60to30v3">48</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oEC60to30wLI">48</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oRRS30to10v3">96</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oRRS30to10wLI">96</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oRRS18to6v3">96</value>
-      <value compset="_DATM.*_SLND.*MPASO" grid="oi%oRRS15to5">96</value>
+      <value compset="_DATM.*_SLND.*MPAS" grid="oi%oQU240">12</value>
```

Comment:
Why was the "O" removed - seems like a source of conflict with MPASLI or MPASA

Reply:
Beats me. @amametjanov ?

Reply:
Those are mpas-o/mpas-cice grids. We would want those coupling intervals for compsets that are datm and either of the mpas ocn/ice components

Reply:
You should consider using MPAS[IO] then.

Reply:
That's fine with me.

Reply:
@jonbob, please make the change suggested by JimE on the ACME side if you want it.